howl/model/workspace.py

```diff
@@ -13,15 +13,18 @@
 from howl.utils.dataclass import gather_dict
 
 
-class WorkspaceSettings(BaseSettings):
+class ModelSettings(BaseSettings):
     model_name: str
     vocab: List[str]
     inference_sequence: List[int]
     token_type: str
     use_frame: bool
-    num_mels: int
-    max_window_size_seconds: int
-    eval_stride_size_seconds: int
+
+
+class WorkspaceSettings(BaseSettings):
+    model: ModelSettings
+    audio_transform: Dict[str, Any] = {}
+    training: Dict[str, Any] = {}
```

Comment:
1. I think these settings must go to https://github.com/castorini/howl/blob/dba1515acc5c49dc15b594ea97228cb2aedb80a2/howl/settings.py
2. I think WorkspaceSettings must be renamed to HowlSettings
3. HowlSettings should have contain every settings in as class variables instead of having audio_transform/trainin explicitly. Just like how it has `model: ModelSettings`

Reply:
Ok, I just reorganized everything to fit inside the existing settings object. I renamed the LazySettingsSingleton to HowlSettings since it seems more clear